identity-iota/src/tangle/encoding.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::Error::CompressionError;
+use crate::tangle::compression_brotli::compress_brotli;
+use crate::tangle::compression_brotli::decompress_brotli;
+use crate::Error;
+
+#[derive(Copy, Clone)]
+pub(crate) enum MessageEncoding {
```

Comment:
Should we rename `MessageEncoding` and `MessageVersion` to `DIDMessageEncoding` and `DIDMessageVersion` to avoid confusion with actual Tangle messages?